docs/GettingStarted.md

```diff
@@ -2361,9 +2361,27 @@ By default, the tracer submits trace data using `Net::HTTP` to `127.0.0.1:8126`,
 
 Some basic settings, such as hostname and port, can be configured using [tracer settings](#tracer-settings).
 
+#### Using the Ethon adapter
+
+The `:ethon` adapter submits traces using [`Ethon`](https://github.com/typhoeus/ethon) over TCP. It is more memory efficient than the default `Net::HTTP` adapter, and it's a recommended replacement for the default adapter.
+Requires `ethon` version `0.8.0` or later to be available in the environment.
+
+```ruby
+# Gemfile
+gem 'ethon', '>= 0.8.0'
+```
+
+```ruby
+Datadog.configure do |c|
+  c.tracer.transport_options = proc { |t|
+    t.adapter :ethon
+  }
+end
+```
```

Comment:
I wonder if this is a bit "too buried". E.g. I understand that we don't want to force this dependency on customers, but as a customer I wouldn't often read through this whole document to check "is there some toggle I can use to make this library work better".

What if we actually had a "recommended setup" section? E.g. a section that we kept up-to-date with all the recommended settings and optional dependencies. That way, customers would have a single very visible spot in the docs where they could see "ok am I using all the latest recommended niceties"?

A bit like rspec has very conservative defaults for compatibility, but when you run `rspec --init` [you get a nice `spec_helper` which lists a bunch of nice stuff you may want to turn on](https://ivoanjo.me/blog/2016/09/25/why-you-should-regenerate-your-spec-helper/). 

(I guess the same thing may end up happening to the profiler part -- we have `ffi` as an optional dependency, so that could be one other thing that we put in there "we recommend you pull in ffi for the profiler").